src/libs/ajax.js

```diff
@@ -182,15 +183,16 @@ const User = signal => ({
 
   getTosAccepted: async () => {
     const url = `${getConfig().tosUrlRoot}/user/response?${qs.stringify(tosData)}`
-    const res = await instrumentedFetch(url, _.merge(authOpts(), { signal }))
-    if (res.status === 403 || res.status === 404) {
-      return false
-    }
-    if (!res.ok) {
-      throw res
+    try {
+      const res = await fetchOk(url, _.merge(authOpts(), { signal }))
+      const { accepted } = await res.json()
+      return accepted
+    } catch (error) {
+      if (error.status === 403 || error.status === 404) {
+        return false
+      }
```

Comment:
I like putting the `else` explicitly, I think it makes it easier for newbies to read.